rest-service/manager_rest/storage/aria_model.py

```diff
@@ -0,0 +1,40 @@
+import aria
+from aria.storage.sql_mapi import SQLAlchemyModelAPI
+
+from .models_base import db
+
+_model_storage = None
+
+
+def initiator():
```

Comment:
The kind of defeats the purpose - the initiator should be serializable. However in our case we don't really need it to be. However we could just pass the engine and session as part of the api_kwargs